http-handler.abi.md

```diff
@@ -0,0 +1,28 @@
+# Types
```

Comment:
this (generated) file only exists to behave as a tripwire, which lets you know if you changed the type while editing the source markdown (as not all changes impact the function definitions)